controllers/account-setting/my-team/get-my-supervisors.controller.js

```diff
@@ -5,6 +5,13 @@ const getMySupervisorsController = async (req, res, next) => {
 	try {
 		const user = req.user;
 
+		if (user.role !== 'CRM_ADMIN' && user.role !== 'CRM_SUPERVISOR') {
```

Comment:
This is redundant and bad practise. use a middleware. Already implemented with routes in notifications or auth